app/src/main/java/ch/epfl/qedit/frontendBackendInterface/RequestLogin.java

```diff
@@ -0,0 +1,19 @@
+package ch.epfl.qedit.frontendBackendInterface;
+
+public class RequestLogin extends Request {
+    private String id;
+    private String password;
+
+    public RequestLogin(String id, String password) {
+        this.id = id;
+        this.password = password;
```

Comment:
I wonder if for the moment if creating the classes RequestLogin and ResponseLogin aren't a little overkill.

Reply:
For now it's overkill as we do not have other requests ...